tests/apps/study/test_study_api.py

```diff
@@ -56,3 +57,77 @@ def test_study_detail_api(db_keys):
         "url": "/study/1/",
         "rob_response_values": [17, 16, 15, 12, 14, 10],
     }
+
+
+@pytest.mark.django_db
+class TestStudyCreateApi:
+    def test_permissions(self, db_keys):
+        url = "/study/api/study/"
+        data = {
+            "reference_id": db_keys.reference_unlinked,
+            "short_citation": "Short citation.",
+            "full_citation": "Full citation.",
+        }
+
+        # reviewers shouldn't be able to create
+        client = APIClient()
+        assert client.login(username="rev@rev.com", password="pw") is True
+        response = client.post(url, data)
+        assert response.status_code == 403
+
+        # public shouldn't be able to create
+        client = APIClient()
+        response = client.post(url, data)
+        assert response.status_code == 403
+
+    def test_bad_requests(self, db_keys):
+        # payload needs to include the required short_citation and full_citation
+        url = "/study/api/study/"
+        data = {"reference_id": db_keys.reference_unlinked}
+        client = APIClient()
+        assert client.login(username="team@team.com", password="pw") is True
+        response = client.post(url, data)
+        assert response.status_code == 400
+        assert {"short_citation", "full_citation"}.issubset((response.data.keys()))
+
+        # invalid references will not be successful
+        data = {
+            "reference_id": "invalid",
+            "short_citation": "Short citation.",
+            "full_citation": "Full citation.",
+        }
+
+        response = client.post(url, data)
+        assert response.status_code == 400
+        assert "Reference ID must be a number." == str(response.data["non_field_errors"][0])
```

Comment:
i generally put expected str after the new data; that way i can read it "i'm going to get this thing and compare it to this expected result"; it reads easier for me. Do you  prefer this way?

Reply:
I prefer it your way, I just haven't kept convention. I'll fix this and keep it in mind going forward!